android/app/src/main/java/org/fossasia/openevent/activities/auth/ChangePasswordActivity.java

```diff
@@ -77,6 +68,12 @@ protected void onCreate(Bundle savedInstanceState) {
         });
     }
 
+    private void getUser() {
+        changePasswordActivityViewModel.getUser().observe(this, user -> {
+            this.user = user;
+        });
+    }
```

Comment:
We can have this function inline in the area called. It is not so big and only used once